.github/workflows/pr.yml

```diff
@@ -113,14 +115,15 @@ jobs:
       shell: bash {0}
       run: |
         cd minikube_binaries
+        docker load -i kic.tar
         mkdir -p report
         mkdir -p testhome
         chmod a+x e2e-*
         chmod a+x minikube-*
         sudo ln -s /etc/apparmor.d/usr.sbin.mysqld /etc/apparmor.d/disable/
         sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
         START_TIME=$(date -u +%s)
-        KUBECONFIG=$(pwd)/testhome/kubeconfig MINIKUBE_HOME=$(pwd)/testhome ./e2e-linux-amd64 -minikube-start-args=--vm-driver=docker  -test.run TestFunctional -test.timeout=30m -test.v -timeout-multiplier=1.5 -binary=./minikube-linux-amd64 2>&1 | tee ./report/testout.txt
+        KUBECONFIG=$(pwd)/testhome/kubeconfig MINIKUBE_HOME=$(pwd)/testhome ./e2e-linux-amd64 -minikube-start-args="--driver=docker --docker-base-image=kic"  -test.run TestFunctional -test.timeout=30m -test.v -timeout-multiplier=1.5 -binary=./minikube-linux-amd64 2>&1 | tee ./report/testout.txt
```

Comment:
same here, if docker image didn't change we should use the current base image, and avoid passing the build image here.

Reply:
resolved as mentioned above